.github/workflows/test-cygwin.yml

```diff
@@ -32,14 +32,8 @@ env:
 jobs:
   build:
     runs-on: windows-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        python-minor-version: [9]
```

Comment:
In December, a Cygwin maintainer said:

> Python 3.12 will be in the near future introduced and we will skip 3.10 
and 3.11.

https://cygwin.com/pipermail/cygwin-announce/2023-December/011437.html

Maybe it's worth retaining this single-entry to make it easier to extend when/if 3.12 shows up?

Reply:
Ok, I've removed that commit.